### ISSUE TITLE
MODULE_NOT_FOUND when running "./bin/run" with installed module

### ISSUE THREAD

**Issue by cameronreynoldson** [oclif/oclif#321]
Do you want to request a *feature* or report a *bug*?
-----------------------------------------------------
Report a *bug*

What is the current behavior?
-----------------------------
1. Create a new Oclif multi-command project.
2. `yarn add @types/n-readlines`
3. add `esModuleInterop: true` to tsconfig.json
4. add `import lineByLine from "n-readlines"` to hello.ts
5. use the imported lineByLine

When I try to run `./bin/run` I get:

```
(node:99195) [MODULE_NOT_FOUND] Error Plugin: mynewcli: Cannot find module 'n-readlines'
Require stack:
- /Users/cameronreynoldson/mynewcli/src/commands/hello.ts
- /Users/cameronreynoldson/mynewcli/node_modules/@oclif/config/lib/plugin.js
- /Users/cameronreynoldson/mynewcli/node_modules/@oclif/config/lib/config.js
- /Users/cameronreynoldson/mynewcli/node_modules/@oclif/config/lib/index.js
- /Users/cameronreynoldson/mynewcli/node_modules/@oclif/command/lib/command.js
- /Users/cameronreynoldson/mynewcli/node_modules/@oclif/command/lib/index.js
- /Users/cameronreynoldson/mynewcli/bin/run
module: @oclif/config@1.13.3
task: toCached
plugin: mynewcli
root: /Users/cameronreynoldson/mynewcli
See more details with DEBUG=*
example description

VERSION
  mynewcli/0.0.0 darwin-x64 node-v13.7.0

USAGE
  $ mynewcli [COMMAND]

COMMANDS
  help  display help for mynewcli
```

What is the expected behavior?
------------------------------
I should not be getting a `module not found` error
Oclif version: latest
OS version: MacOS Catalina 10.15.1

**Comment by cameronreynoldson** [comment]
This seems to be an issue with the `esModuleInterop` flag, but why? I thought this was already fixed in #179

**Comment by cameronreynoldson** [comment]
I went digging and tried to find libraries that used Oclif and Typescript and also external node modules, and noticed that one that worked didn't have any `@types/moduleName` in the `dependencies` section of their `package.json` config.

So to (temporarily) fix this, I removed the `@types/moduleName` package from my project, installed the default (just `moduleName`), and created my own Typescript declaration file for the module. It seems to work fine with this, but this seems a bit wild to have to do for every node module myself?
